src/frontend-common/common_host_interface.cpp

```diff
@@ -983,18 +1004,20 @@ void CommonHostInterface::UpdateControllerInputMap(SettingsInterface& si)
         si.GetStringList(category, TinyString::FromFormat("Button%s", button_name.c_str()));
       for (const std::string& binding : bindings)
       {
-        std::string_view device, button;
-        if (!SplitBinding(binding, &device, &button))
-          continue;
-
-        AddButtonToInputMap(binding, device, button, [this, controller_index, button_code](bool pressed) {
-          if (!m_system)
-            return;
-
-          Controller* controller = m_system->GetController(controller_index);
-          if (controller)
-            controller->SetButtonState(button_code, pressed);
-        });
+        DeviceSubBindingList splits;
```

Comment:
This should already be a list - see the `GetStringList()` above. I might not have hooked it up on the other end though...